src/main/java/org/candlepin/controller/ContentAccessManager.java

```diff
@@ -524,15 +553,14 @@ private Set<X509ByteExtensionWrapper> prepareV3ByteExtensions(org.candlepin.mode
     }
 
     private byte[] createContentAccessDataPayload(Consumer consumer,
-        Map<org.candlepin.model.Content, Boolean> ownerContent, PromotedContent promotedContent)
+        Map<String, ProductContent> activeContent, PromotedContent promotedContent)
         throws IOException {
         log.info("Generating SCA payload for consumer \"{}\"...", consumer != null ? consumer.getUuid() :
             "unknown");
         Product engProduct = new Product()
             .setId("content_access")
-            .setName(" Content Access");
-
-        ownerContent.forEach(engProduct::addContent);
+            .setName(" Content Access")
```

Comment:
non-review question: Is the space in front of "Content Access" intentional here? What was that for?

Reply:
Looking at the log, it's been like that since SCA was first implemented. I thought maybe this was done as a 'UX' enhancement to make sure it looks when someone prints the cert using `rct cat-cert`, or maybe the traditional entitlement certs also had a space appended before product names, but neither seem to be the case. So it looks accidental.
However, I'm not sure if changing it now is a good idea, because there may be users that are parsing this (for whatever reason), which would break.

This made me think though, do we want to set a different product id/name for the anonymous cert? Something like 'Anonymous Content Access', or 'Cloud Offering Content Access' (there is maybe a better name I can't think of right now). It may make it easier for testing/debugging purposes, if there's such an indication.

Reply:
cc: @joshmalbrecht

Reply:
I like that idea of giving some indication that this is an anonymous Certificate for the reasons you listed! I was taking a look at Authorizer and I don't think that would cause any issues in their parsing of the certificate, but I'm not super familiar with that code.